config/jobs/containerd/cri/containerd-cri-presubmit-jobs.yaml

```diff
@@ -53,7 +53,7 @@ presubmits:
           &&
           /workspace/scenarios/kubernetes_e2e.py
           --deployment=node
-          --gcp-project=cri-c8d-pr-node-e2e
+          --gcp-project=cri-containerd-node-e2e
```

Comment:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml#L63

did you mean to hit this file ^

Reply:
whoops fixed.